src/managed/OpenLiveWriter.PostEditor/Video/VideoProvidersB2.xml

```diff
@@ -79,28 +79,28 @@
     <serviceName>YouTube</serviceName>
     <editorFormat xpp:if="version lt 15.4.3104.0816">
       <![CDATA[
-		<embed src="http://www.youtube.com/v/{videoId}" type="application/x-shockwave-flash" wmode="transparent" width="{width}" height="{height}"></embed>
+		<embed src="http://www.youtube.com/embed/{videoId}" wmode="transparent" width="{width}" height="{height}"></embed>
```

Comment:
We should maybe consider using the same YouTube embedding syntax as provided through the Share button on YouTube.

```
<iframe width="560" height="315" src="https://www.youtube.com/embed/X-a56UVN0cc" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Reply:
Yes, Nick, even though there is not much difference in case of video embedding but its better to use <iframe> than <embed>, I also thought to change it to iframe but had doubts about whether to change existing things or not. Better we'll change it to <iframe> :)